src/Internal/Processor.php

```diff
@@ -805,12 +810,15 @@ private function handleHandshake($packet)
     /** @see 14.6.4.1.2 LOCAL INFILE Request */
     private function handleLocalInfileRequest($packet)
     {
```

Comment:
I'm not happy with this asyncCall. On failure this will throw into the loop. Should probably fail the current active query instead (so that the potential exception like file not found can be caught where the query is issued).

This is the only problem I see, can be merged afterwards.

Reply:
Fixed